dev-tools/src/lambdas/emptyS3Buckets/handler.ts

```diff
@@ -2,13 +2,21 @@ import { CloudFormationCustomResourceEvent, Context } from 'aws-lambda'
 import { emptyS3Bucket } from './emptyS3Bucket'
 import { listS3Buckets } from './listS3Buckets'
 import axios from 'axios'
-import { initialiseLogger, logger } from '../../utils/logger'
+import {
+  appendKeyAttributeDataToLogger,
+  initialiseLogger,
+  logger
+} from '../../utils/logger'
 
 export const handler = async (
   event: CloudFormationCustomResourceEvent,
   context: Context
 ): Promise<void> => {
   initialiseLogger(context)
+
+  const stackId = event.StackId
+  appendKeyAttributeDataToLogger({ stackId })
```

Comment:
This is going to cause us confusion at some point, because you're appending keys to the logging context that aren't removed in initialiseLogger. I'd either take them out altogether (chances are they're maybe not that useful) or add something to remove them in initialise.